.pylintrc

```diff
@@ -187,7 +187,7 @@ max-nested-blocks=5
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=80
+max-line-length=79
```

Comment:
Ok, už to čtu, ráno se mi to nějak nepodařilo vygooglit.. Je to teda ale imho dost demence..